### PR TITLE
API Create Project

### PR DESCRIPTION
**Pull request by quocquann**
## Related Tickets
[- ticket redmine](https://edu-redmine.sun-asterisk.vn/issues/71177)

## HOW
create project api using django_rest_framework

### CODE REVIEW

**Comment by quocquann** [comment]
ready

**Comment by quocquann** [comment]
ready

**Comment by quocquann** [comment]
image

File: api/serializers.py [hunk 1]
```diff
@@ -5,6 +5,8 @@
 from rest_framework.validators import UniqueValidator
 
 from app.utils.helpers import check_token
+from app.models import Project
+import datetime
```
Comment (manhnv-1320) [inline api/serializers.py:9]: import của python lên đầu

File: api/urls.py [hunk 2]
```diff
@@ -1,6 +1,7 @@
 from django.urls import path
 from drf_spectacular.views import SpectacularAPIView, SpectacularSwaggerView
 from rest_framework_simplejwt.views import TokenObtainPairView, TokenRefreshView
+from . import views
```
Comment (manhnv-1320) [inline api/urls.py:4]: xóa bớt

File: api/urls.py [hunk 3]
```diff
@@ -15,4 +16,5 @@
     path("token/refresh", TokenRefreshView.as_view(), name="token_refresh"),
     path("signup", views.SignUp.as_view(), name="signup"),
     path("verify/<str:uidb64>/<str:token>", views.Verify.as_view(), name="verify"),
+    path("projects", views.create_project),
```
Comment (manhnv-1320) [inline api/urls.py:19]: thiếu name

**Review by manhnv-1320 — APPROVED** [review]

**Comment by quocquann** [comment]
ready

**Review by huongnt-2545 — APPROVED** [review]
merged

**Pull request merged**